src/App.tsx

```diff
@@ -1,8 +1,39 @@
-import React from 'react';
-
-export const App: React.FC = () => (
-  <div className="App">
-    <p>Nothing was pressed yet</p>
-    <p>The last pressed key is [Enter]</p>
-  </div>
-);
+import { Component } from 'react';
+
+type State = {
+  pressedKey: string,
+};
+
+export class App extends Component<{}, State> {
+  state = {
+    pressedKey: '',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.keyListener);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.keyListener);
+  }
+
+  keyListener = (event: KeyboardEvent) => {
+    this.setState({ pressedKey: event.key });
+  };
+
+  render() {
+    const { pressedKey } = this.state;
+
+    return (
+      <div className="App">
+        <p>Nothing was pressed yet</p>
+        {pressedKey && (
+          <p>
+            The last pressed key is
+            {pressedKey}
```

Comment:
```suggestion
            {`The last pressed key is ${pressedKey}`}
```
И лучше написать таким образом, чтобы после is был пробел:)
Потому что в React перенос строки - не пробел.